waku/node/peer_manager/peer_manager.nim

```diff
@@ -75,7 +75,7 @@ type
     initialBackoffInSec*: int
     backoffFactor*: int
     maxFailedAttempts*: int
-    storage: PeerStorage
+    storage*: PeerStorage
```

Comment:
For testing in `app.nim`. Wanted to import the file instead but there's already one import for the same purpose and they have conflicts.